### DISCUSSION TITLE
Run streamlink in GitHub actions

### DISCUSSION THREAD

**Discussion by guiworldtv2** [streamlink/streamlink#4611] [Q&A] [answered]
Someone know if its possible execute streamlink inside github actions?
my plan its do a m3u8 link to my iptv using streamlink, doing all in the github and give me only the m3u8 of some transmitions
its possible? i'm bad with languages programations

**Comment by gravyboat** [comment, marked as answer]
Even if this was possible we wouldn't support it or provide suggestions as it's a clear abuse of GitHub's services. Run your own server.
